src/lib/API/api.js

```diff
@@ -66,9 +68,22 @@ export class APIService {
 
   client: AxiosInstance
 
+  sharedClient: AxiosInstance
+
   mauticJS: any
 
   constructor(jwt = null) {
+    const shared = axios.create()
```

Comment:
why do we need another client?

Reply:
we used direct `axios()` call in a few places. i suppose for skip out interceptors and do not send headers filled up for our backed
i've added this.sharedClient which have just a simple interceptor returning response.data as a result to simplifiy axios() calls